arches_component_lab/views/api/relatable_resources.py

```diff
@@ -12,12 +12,21 @@
 
 class RelatableResourcesView(View):
     def get(self, request, graph, node_alias):
-        node = Node.objects.get(
-            alias=node_alias,
-            graph__slug=graph,
-            graph__is_active=True,
-            graph__publication__isnull=False,
+        node = (
```

Comment:
style nit: multi-line ternary should be refactored

Reply:
Yeah I think @bferguso said [here](https://github.com/archesproject/arches-component-lab/pull/49#issuecomment-2886930812) he was working on that to have the if/else just cover one statement, e.g. chaining another `.filter()` or ORing another `models.Q` is my guess...